srcpkgs/radicale2/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'radicale2'
+pkgname=radicale2
+version=2.1.10
+revision=1
+archs=noarch
+wrksrc="Radicale-${version}"
+build_style=python3-module
+pycompile_module="radicale"
+conf_files="
```

Comment:
please put conf_files and make_dirs brllow the big variable block  toghether with system_accounts

Reply:
like this?